docs/extras/autorcm.md

```diff
@@ -18,6 +18,7 @@
 	- Please keep in mind that the console can no longer boot on its own, so you'll need a PC, phone, or other payload injector to start the console after letting the console fully shut off. (AutoRCM by itself is considered a "tethered jailbreak").
 		Note: Sleep Mode is not considered "powering off/on" the console. Sleep Mode will still work as intended and is fully unchanged with AutoRCM.
 	- When completely discharged, your Switch will take a very long time to charge while in RCM. To solve this, charge the console for approximately twenty minutes before booting into Hekate and selecting a boot option. After it's done booting, the console will now charge at a normal speed with a battery icon in the corner.
+	-When a system update is completed while stock firmware is running, AutoRCM becomes disabled so you need to turn it on again. AutoRCM can be preserved by updating while CFW is active."
```

Comment:
For consistency, probably add a space after the dash and before the `when`.

Reply:
Alright